.github/connector_dependency_comment_template.md

```diff
@@ -0,0 +1,24 @@
+<!--- this comment is for `report-connectors-dependency.yml` identification, do not remove -->
```

Comment:
I like this template pattern!  Perhaps make a `.github/comment_templates` directory if you want to use this kind of thing again

Reply:
Good point, I'll go ahead and make that now, seems like a lot of our slash commands could make use of templates in the future if we want to make any improvements to those